priv/mssql2012.sql

```diff
@@ -474,6 +474,31 @@ GO
 CREATE INDEX i_inbox_ts ON inbox(luser, lserver, timestamp);
 GO
 
+CREATE TABLE dbo.pubsub_nodes (
+    nidx BIGINT           NOT NULL,
+    p_key NVARCHAR(250)   NOT NULL,
+    name NVARCHAR(250)    NOT NULL,
+    type NVARCHAR(250)    NOT NULL,
+    owners NVARCHAR(max)  NOT NULL,
+    options NVARCHAR(max) NOT NULL,
+    CONSTRAINT PK_pubsub_nodes PRIMARY KEY CLUSTERED(
```

Comment:
With such order in primary key (applies to all schemas), queries using only `p_key` and/or `name` won't use the index. I think what we need here is `p_key, name, nidx` with possibly separate index for `nidx` alone.

Reply:
Good catch!